### PR TITLE
Add documentation for Nutanix additionalCategories feature

### PR DESCRIPTION
**Pull request by yannickstruyf3**
*Issue #, if available:*
N/A
*Description of changes:*
Add documentation for the the Nutanix additionalCategories feature
*Testing (if applicable):*
N/A
*Documentation added/planned (if applicable):*
Yes
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by abhinavmpandey08 — APPROVED** [review]

File: docs/content/en/docs/getting-started/nutanix/nutanix-spec.md [hunk 1]
```diff
@@ -278,7 +281,15 @@ Name of the project
  
 ### project.uuid (`name` or `UUID` required)
 UUID of the project
+
+### additionalCategories	(optional)
+Reference to a list of existing categories to be assigned to virtual machines.
```
Comment (chrisnegus) [inline docs/content/en/docs/getting-started/nutanix/nutanix-spec.md:286]: Could you identify these as Nutanix categories and provide a link? Something like this: https://portal.nutanix.com/page/documents/details?targetId=Nutanix-Calm-Admin-Operations-Guide-v3_7_0:nuc-blueprints-tags-categories-overview-c.html

**Review by chrisnegus — COMMENTED** [review]
Just one comment.

File: docs/content/en/docs/getting-started/nutanix/nutanix-spec.md (hunk 1 above)
Comment (yannickstruyf3) [reply docs/content/en/docs/getting-started/nutanix/nutanix-spec.md]: Updated the documentation.

**Review by abhinavmpandey08 — APPROVED** [review]
/approve

**Pull request merged**